src/components/Button/index.stories.tsx

```diff
@@ -25,12 +27,55 @@ export const Outline: Story = {
   args: {
     variant: "outline",
     children: "Button",
+    // colorScheme: "primary",
```

Comment:
@Kashyap2701 Can we remove this comment?

Reply:
Yes, we can